docs/UC_CV_3ElectrodeCell.md

```diff
@@ -0,0 +1,40 @@
+Use Case: Cyclic Voltammetry on the JUAMI Potentiostat
+Intended User: Undergraduate Researcher at a University in Africa with Introductory Knowledge of Electrochemistry and Cyclic Voltammetry. The instructions in this UC assume that the student already has an electrochemical cell set up and ready for testing (working electrode, counter electrode, and possibly reference electrode are all in a proper beaker).
+As a test case - Mike will perform these instructions as someone with knowledge of electrochemistry but little knowledge of Python.
+
+Experimental Setup:
+
+1. Mike plugs the USB cable into the computer and the potentiostat
+1. Mike attaches the red wire to the working electrode (material of interest)
+1. Mike attaches the black clip to the counter electrode
+1. Mike attaches the white clip to the reference electrode if he is using one, or he attaches it to the counter electrode for a two electrode test
+
+Program Setup:
+1. Mike opens the Pytentiostat script (the script is incomplete and requires filling in of user-defined information)
```

Comment:
not sure what Mike sees when he "opens the script".  Does he double click on an icon on the desktop, or type ``python scriptname`` on a command line?  Then it seems to be asking for user-defined info...so it looks like a gui form?

If a gui is the end-goal we can also write two UCs, one without the gui and one with the gui.  The second UC would read something like:
UC2:
As UC1 but Mike double-clicks an icon on the desktop to open a gui and fills in the information into a gui form.

In this way we can build the code for UC1 and test it, and then build the gui on top.  Separating functionality and interface is always a good idea.